src/ucp/core/ucp_worker.h

```diff
@@ -338,6 +338,10 @@ typedef struct ucp_worker {
                                                            * used to minimize call of ucs_get_time */
         size_t                       round_count;         /* Number of rounds done */
     } keepalive;
+    /* Number of requests to create endpoint */
```

Comment:
`Number of successful requests to create endpoint` to clarify it counts only successful ep creations

Reply:
Actually, this is a total number of requests to create EP.

Reply:
ic